content/syllabuses/data-sci.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Data Science"
+---
+
+- Introduction to NCIT curriculum
```

Comment:
Everything in this file will result in a card on the board that is highlighted.

Then everything not on this page that is direcrly or indirectly required by something on this page will be on the board as a card that is not highlighted.

If we highlight everything then everything looks like a big serious goal and we might as well highlight nothing.  Ideally we would highlight major milestones.

Eg: simple-calc part 1 really shouldn't be on this page because it's a helloworld.  It is a pre-cursor to the string calculaor. The string calculator is a worthy goal. 

Then if there are any two lines like:

```
- {{% contentlink "topics/important-thing" %}}
- {{% contentlink "projects/important-thing" %}}
```

where the topic is already a precursor of the project , then the topic should be removed

```
- {{% contentlink "projects/important-thing" %}}
```

In general putting a topic on this page should be done with caution because they are generally hard prerequisites to projects that should be put on this page.

Please take a look at data-eng.md.